TemporaryOysters/bangazon-api
# Me product serializer

##Pull Requests



##Descriptions:

* Please describe what issue this PR solves
These commits create a product serializer, product view, and urls necessary for user, group and product.





I updated our User views to viewsets for efficiency. This eliminates the need for separate list and detail views. I also added a Groups viewset for grouping Users.


With these additions, user should be able to login and navigate through the viewsets listed above.

##Steps to Test:


1. Git Fetch
2. Open this branch me-productSerializer
⋅⋅*python manage.py migrate
⋅⋅*python manage.py runserver
3. Go to http://localhost:8000
⋅⋅*login using our superuser Oyster
⋅⋅*navigate though views, testing User, Groups, and Product



##System configuration.


* Are there 3rd party libraries that need to be installed?
No additional libraries are necessary

* Any command line utilities to run?
See above


In the UI, you should see views for User, Group and Product. From the main login page, you should be able to click on hyperlinks to navigate to each of these. There have been no products added, but the necessary fields are there.
